app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -273,6 +276,24 @@ class AdvancedFilter extends React.Component {
     });
   };
 
+  // Reset modal when cancelled
+  cancel = () => {
+    // check if there was a filter applied
+    const appliedFilterId = localStorage.getItem(`SaraFilter`);
+    const appliedFilter = this.state.savedFilters.find(filter => filter.id === parseInt(appliedFilterId));
+
+    // update state accordingly
+    const applied = !!appliedFilterId;
+    const activeFilter = applied ? appliedFilter : null;
+    const activeFilterOptions = applied ? appliedFilter.contents : [];
```

Comment:
While testing, I got an `Uncaught TypeError: appliedFilter is undefined` error that traced back to this line when selecting the cancel button. Here are the steps to reproduce:

1. Apply a saved filter.
2. In the Advanced filter drop down, select "Clear Current Filter".
3. Open up a saved filter & click the Cancel button.